### PR TITLE
precourse projects

### PR DESCRIPTION
**Pull request by Vacadery**

### CODE REVIEW

File: staff/emma-cowdery/precourse/Tema-1/project1.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+		console.log('result = [' + num1 + ' + ' + num2 + ' = ' + resultAdd() + ', ' + num1 + ' - ' + num2 + ' = ' + resultSub() + ', ' + num1 + ' * ' + num2 + ' = ' + resultMult() + ', ' + num1 + ' / ' + num2 + ' = ' + resultDiv() + ']');
+		
+	} else {
+		console.log('You must put in a number');
+	};
+
+};
+
+calculate(2, 3);
```
Comment (robert-z) [inline staff/emma-cowdery/precourse/Tema-1/project1.js]: Add newline at end

File: staff/emma-cowdery/precourse/Tema-1/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# CALCULADORA
+
+![calculator logo](calculator-icon.png)
+
+## - INTRODUCCIÓN
```
Comment (robert-z) [inline staff/emma-cowdery/precourse/Tema-1/README.md:5]: I think the `-` is not necessary

File: staff/emma-cowdery/precourse/Tema-2/project2.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        for(let i = 6; i < flights.length; i++) {
+            console.log(flights[i].to);
+        };
+    };
+
+    console.log(lastFive());
+
+};
+
+console.log(airline());
```
Comment (robert-z) [inline staff/emma-cowdery/precourse/Tema-2/project2.js]: Add newline at end

File: staff/emma-cowdery/precourse/Tema-2/project2.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            } else {
+                console.log(`El vuelo con origen: ${flights[i].from}, y destino: ${flights[i].to} tiene un coste de ${flights[i].cost} y realiza escala.`)
+            };
+        };
+    };
+
+    console.log(vuelos());
+
+    function avg() {
+        sum = 0;
```
Comment (robert-z) [inline staff/emma-cowdery/precourse/Tema-2/project2.js:35]: Add `let` instruction to `sum` variable

File: staff/emma-cowdery/precourse/Tema-1/project1.js (hunk 1 above)
Comment (Vacadery) [reply staff/emma-cowdery/precourse/Tema-1/project1.js]: line added

File: staff/emma-cowdery/precourse/Tema-1/README.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+Calculadora que da el resultado de la suma, resta, multiplicación y diviión de dos numeros dados.
+
+## - COMO USAR
+
+Al llamar la función (*calculate*) introduces los dos numeros deseados en ella.
+
+### - **Ejemplo:**
```
Comment (robert-z) [inline staff/emma-cowdery/precourse/Tema-1/README.md:13]: The `**` aren't necessary in this case because it is a heading 3 (bold style is include)

**Pull request merged**